### DISCUSSION TITLE
Verticals tab not indicating selected value when using ?v=TabName or ?v=TabValue

### DISCUSSION THREAD

**Discussion by boing0** [microsoft-search/pnp-modern-search#3449] [Q&A] [answered]
I think I've got verticals set up correctly -- they refine the results as expected when I click on the tabs. When I use the query string v=TabName the results are also pre-refined for that tab value. But the selected tab doesn't indicate the query string value with the blue underline, it always shows the first tab as selected. I'm using {searchTerms} AND Site.Id:{verticals.value} in the Results Query template. I have verticals configured not as hyperlinks, which I thought was correct? I also tried v=TabValue (the Site.Id value) but that didn't work. Any idea what I'm doing wrong? I'm using 4.10.1.

Query string in use, results are correct but the right tab isn't selected, it's showing the first tab.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/8752412/195c7766-e936-43e9-ad29-78bf695c43ff)

![image](https://github.com/microsoft-search/pnp-modern-search/assets/8752412/7ac7022e-6fd9-490e-be3a-8a528a1c722c)

**Comment by nygarm1** [comment]
I am experiencing the same behavior.  I suspect a bug was introduced in version 4.10.1 such that, when using the v parameter for vertical selection, the verticals web part only ever displays the default (first) vertical. Try installing the previous version (4.9.3).

**Comment by wobba** [comment]
I created a bug for this if it worked fine in v4.9 https://github.com/microsoft-search/pnp-modern-search/issues/3452

**Comment by patrikhellgren** [comment, marked as answer]
This is now fixed in version [4.11 Pre-release](https://github.com/microsoft-search/pnp-modern-search/releases/tag/4.11.0)
